README.md

```diff
@@ -117,4 +171,6 @@ $ rake release
 
 ## Contributors
```

Comment:
We should probably just dump both authors and contributors here. So add me, you, etc... anybody with an authored commit?

Reply:
👍